frontend/src/components/Search.jsx

```diff
@@ -8,21 +9,30 @@ import Error from 'components/Error';
 import Loading from 'components/Loading';
 
 class Search extends React.Component {
+  constructor(props) {
+    super(props);
+    this.searchDebounced = _.debounce(this.props.search, 300);
```

Comment:
Why does this function call need to be debounced? Not clear to me here.

Reply:
Could use a comment.

Reply:
Also, if we now need lodash as part of Conductor, it should be added via yarn to the package.json file. Right now it just happens to be a subdependency, but it's a direct dependency so we should be explicit about it.

Reply:
Oh, we chose to use debounce the search to avoid throttling the api when searching for commits.

Added comment.

Explicitly added lodash using yarn